src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -19,6 +19,9 @@ This provider is independent from `MeetingProvider`. You can put `VoiceFocusProv
 
 ```
 
+You should see "Amazon Voice Focus is supported." in the dev tool of the broswer if your device is sufficient for Amazon Voice Focus feature,
```

Comment:
Suggest adding below link for more information on support:
https://aws.github.io/amazon-chime-sdk-js/modules/amazonvoice_focus.html#can-i-use-amazon-voice-focus-in-my-application

Should we re-frame the sentence to, "You should see 'Amazon Voice Focus is supported.' or 'Amazon Voice Focus is not supported.' console logs in your browser developer tools based on Amazon Voice Focus support on your machine and browser. For more information, check Amazon Chime SDK for JavaScript's Amazon Voice Focus [guide](Link to guide)."

Reply:
Good point, will add the link for reference